macros/sql/star.sql

```diff
@@ -9,6 +9,7 @@
     {#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
     {%- if not execute -%}
         {{ return('') }}
+        {{ return('*') }}
```

Comment:
Only the first `return` block would be hit, so your new `'*'` would never make it back! 

```suggestion
        {{ return('*') }}
```